drivers/modem/ublox-sara-n310.c

```diff
@@ -835,6 +897,148 @@ static bool offload_is_supported(int family, int type, int proto)
 NET_SOCKET_REGISTER(ublox_sara_n310, AF_UNSPEC, offload_is_supported,
 		    offload_socket);
 
+/*
+ * Pin functions.
+ *
+ * The SARA-N310 forces the power pin low to achieve the following:
+ * min 1s - max 2.5s: trigger module switch-on
+ * min 1s - max 2.5s: trigger module wake-up from PSM
+ * min 2.5s: trigger module switch-off
+ */
+static int turn_on_module(void)
+{
+	int ret, timeout = 0;
```

Comment:
You know that here only one variable is initialized ,
If you want to do it in one line
```
int ret=0, timeout = 0;
```

I am always in favor though of declaring vars in separate lines.